### PR TITLE
Add geo array dumper

### PR DESCRIPTION
**Pull request by norkunas**
#504

### CODE REVIEW

**Comment by drupol** [comment]
I like that !

File: src/Geocoder/Dumper/GeoArray.php [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+/**
+ * @author Jan Sorgalla <jsorgalla@googlemail.com>
+ */
+class GeoArray implements Dumper
+{
+    /**
+     * {@inheritDoc}
+     */
+    public function dump(Address $address)
```
Comment (Nyholm) [inline src/Geocoder/Dumper/GeoArray.php]: Should be `Location`

**Comment by Nyholm** [comment]
I think you did `merge` instead of `rebase`.. 

To recover: 
```
git checkout master
git reset --hard origin/master
git checkout -b array-dumper-2
git cherry-pick 8c651f9
git push -u norkunas array-dumper-2
```

(You could aslo force push to this branch.)

**Comment by norkunas** [comment]
Sorry, fixed in new PR

**Pull request closed without merging**